database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,12 +15,14 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
-            $table->string('name');
+            $table->string('name')->unique();
```

Comment:
maybe if you had a username(nick) unique that would make sense. But for name, I do not think that being unique is really something necessary. For example, Paul Smith there are more people with that name... so making this attribute unique is a bit limiting I think.